data/profiles/demo-auth.yaml

```diff
@@ -0,0 +1,474 @@
+
+# Traffic management feature
+trafficManagement:
+  enabled: true
+  components:
+    namespace: istio-control
+
+    pilot:
+      common:
+        k8s:
+          env:
+          - name: GODEBUG
+            value: "gctrace=1"
+          hpaSpec:
+            maxReplicas: 5
+            minReplicas: 1
+            scaleTargetRef:
+              apiVersion: apps/v1
+              kind: Deployment
+              name: istio-pilot
+            metrics:
+              - type: Resource
+                resource:
+                  name: cpu
+                  targetAverageUtilization: 80
+          replicaCount: 1
+          readinessProbe:
+            handler: {}
+# FIME: what's the problem here?
+#            httpGet:
+#              path: /ready
+#              port: 8080
+            initialDelaySeconds: 5
+            periodSeconds: 30
+            timeoutSeconds: 5
+          resources:
+            requests:
+              cpu: 10m
+              memory: 100Mi
+
+        values:
+          image: pilot
+          autoscaleEnabled: fase
+          traceSampling: 100.0
+          configNamespace: istio-config
+          keepaliveMaxServerConnectionAge: 30m
+          configMap: true
+          ingress:
+            ingressService: istio-ingressgateway
+            ingressControllerMode: "OFF"
+            ingressClass: istio
+          telemetry:
+            enabled: true
+          policy:
+            enabled: false
+          useMCP: true
+
+    proxy:
+      common:
+        values:
+          image: proxyv2
+          clusterDomain: "cluster.local"
+          resources:
+            requests:
+              cpu: 10m
+              memory: 40Mi
+          concurrency: 2
+          accessLogFile: "/dev/stdout"
+          accessLogEncoding: TEXT
+          logLevel: warning
+          componentLogLevel: "misc:error"
+          dnsRefreshRate: 300s
+          privileged: false
+          enableCoreDump: false
+          statusPort: 15020
+          readinessInitialDelaySeconds: 1
+          readinessPeriodSeconds: 2
+          readinessFailureThreshold: 30
+          includeIPRanges: "*"
+          autoInject: enabled
+          tracer: "zipkin"
+
+# Policy feature
+policy:
+  enabled: true
+  components:
+    namespace: istio-policy
+    policy:
+      common:
+        k8s:
+          replicaCount: 1
+          hpaSpec:
+            maxReplicas: 5
+            minReplicas: 1
+            scaleTargetRef:
+              apiVersion: apps/v1
+              kind: Deployment
+              name: istio-pilot
+            metrics:
+              - type: Resource
+                resource:
+                  name: cpu
+                  targetAverageUtilization: 80
+          resources:
+            requests:
+              cpu: 10m
+              memory: 100Mi
+          env:
+            - name: GODEBUG
+              value: "gctrace=1"
+        values:
+          image: mixer
+          autoscaleEnabled: false
```

Comment:
this looks incompatible with the hpaSpec above. Probably set it to true.

Reply:
Currently, autoscaler or hpa is disabled for demo and demo-auth profiles.
However, we can't specify whether to enable autoscaler in the new platform level installation API, so I have to add `autoscaleEnabled: false` to disable autoscaler for demo profile.

I thought we should remove `hpaSpec` in k8s section.

Reply:
Yes, in that case please remove hpaSpec.

Reply:
Sure.